### ISSUE TITLE
Pythonista should be an "Open In" target again

### ISSUE THREAD

**Issue by dgelessus** [omz/Pythonista-Issues#468]
Now that importing Python scripts from outside the app is a fully supported feature in the beta, I think Pythonista should also be an "Open In" target again. IIRC the only reason why it was removed was because it could be used to import code. Now that this is okay again, there's no reason not to re-add that capability.

**Comment by wgaylord** [comment]
It sort of is. Under Run Pythonista script there is an Import File option.

**Comment by dgelessus** [comment]
I know, but that was only as a workaround to allow importing anything *except* Python scripts. (This wasn't possible with the regular "Open In" system.) In the latest beta (311015) that option also works for Python files again, but it's still an extra tap away (plus loading time for the extension) than a regular "Open In".

**Comment by omz** [comment]
The current beta is already an "open in" target for .py and other plain text files. Does that not work, or would you suggest adding support for other file types (I'm kinda against that).

**Comment by dgelessus** [comment]
@omz Interesting. I tried "sharing" a Python file from within Pythonista, and Pythonista didn't appear as an option, so I thought it didn't work. Now that I think about it, the problem is really obvious. Yes, it works with Python, text and Markdown files if I open from another app.

I still think other file types would be useful too, for example ZIP files, images, sounds and PYUIs. I suppose you could add those specific file types. However accepting any type is useful for when it's something that Pythonista doesn't understand, but a script does (like a data file, or a tar.gz for use in Stash, etc.). Yes, I know the share extension can be used, but IMHO just because a file type is "unknown" doesn't mean it should be harder to import into Pythonista.

**Comment by zrzka** [comment]
I agree with Ole. There're other ways how to share all kind of files with Pythonista and Open in should support files that are editable, runnable, ... by Pythonista only. Open in menu will be bloated if every app will just say - okay - I can open whatever you give me. I also know that I can disable / enable apps in this menu, but it's kinda crazy to go through this long list of apps.
